CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ endif()
 #-----------------------------------------------------------------------------
 # OpenGL library as a stand-alone renderer not wrapping D3D
 if ( ( (OPENGL_FOUND AND GLEW_FOUND) OR OPENGLES2_FOUND ) AND (SDL2_FOUND OR ANDROID))
-	list(APPEND RenderManager_SOURCES osvr/RenderKit/RenderManagerOpenGL.cpp osvr/RenderKit/RenderManagerOpenGL.h osvr/RenderKit/RenderManagerOpenGLC.cpp)
```

Comment:
The check for insertion of the OpenGLATW appears not to be strict enough.  The code fails to compile on Windows due to the following:
f:\taylorr\personal\work\consulting\sensics\osvr\src\osvr-rendermanager\osvr\renderkit\RenderManagerOpenGLATW.h(36): fatal error C1083: Cannot open include file: 'EGL/egl.h': No such file or directory